Makefile

```diff
@@ -4,6 +4,8 @@ REPO ?= kubevirt
 IMAGE_TAG ?= latest
 IMAGE_GIT_TAG ?= $(shell git describe --abbrev=8 --tags)
 IMG ?= $(REPO)/kubemacpool
+OCI_BIN ?= $(shell if hash podman 2>/dev/null; then echo podman; elif hash docker 2>/dev/null; then echo docker; fi)
```

Comment:
Why don't we use `determine_cri_bin` here ?

Reply:
because it is in hack/common.sh, so we will need to make this file standalone runnable (or source it unless it does more things)

btw i am checking both here and determine_cri_bin to be simpler, logic like
`export OCI_BIN ?= $(shell if podman ps >/dev/null 2>&1; then echo podman; elif docker ps >/dev/null 2>&1; then echo docker; fi)`

Reply:
ack, let's keep it simple.